src/components/buttons/button-accessibility-image-alt.jsx

```diff
@@ -14,23 +14,16 @@ import ReactDOM from 'react-dom';
  *
  */
 class ButtonAccessibilityImageAlt extends React.Component {
+	static contextType = EditorContext;
+
 	static key = 'AccessibilityImageAlt';
 
-	static propTypes = {
-		/**
-		 * The editor instance where the component is being used.
-		 *
-		 * @instance
-		 * @memberof ButtonAccessibilityImageAlt
-		 * @property {Object} editor
-		 */
-		editor: PropTypes.object.isRequired,
-	};
+	static propTypes = {};
```

Comment:
Just out of curiosity, what's the benefit of an empty `proptypes` definition? Looks like we're not validating `propTypes` in many places... should we file an issue to track down what we want to do about it?

Reply:
I don't think there's any real "benefit". I just wanted to make the minimal change, which was removing the property. You probably won't be surprised to know that I would rather see us using static types and just get rid of runtime prop-type checking.

Reply:
Followed up in #1084.